sqlserver/tests/conftest.py

```diff
@@ -104,6 +112,13 @@ def sqlserver_can_connect():
                 'Always On Availability Groups connection with primary database established for secondary database',
             )
         ]
+    else:
```

Comment:
Needed this because current `conditions` does not give enough time for `master` db to finish backing up.